src/user/actions/update-my-password.action.ts

```diff
@@ -0,0 +1,27 @@
+import { ApiResponse, Fetcher } from '../../common/api';
+import { endpoints } from '../api/endpoints';
+import { IUpdatePasswordDto } from '../types';
+import { errorFactory, UserErr } from '../errors';
+
+export const updateMyPassword = async (
+  fetcher: Fetcher,
+  data: Partial<IUpdatePasswordDto>,
+): Promise<IUpdatePasswordDto> => {
```

Comment:
Attention à la réponse de la requête, ce n'est pas nécessairement la même chose que le payload d'entrée. Cette route retourne juste un booléen.
```suggestion
): Promise<boolean> => {
```